report_templates/host_-_all_installed_packages.erb

```diff
@@ -0,0 +1,24 @@
+<%#
+name: Host - All Installed Packages
+snippet: false
+template_inputs:
+- name: hosts
+  required: false
+  input_type: user
+  advanced: false
+  value_type: plain
+  resource_type: Katello::ActivationKey
+  hidden_value: false
+model: ReportTemplate
+%>
+<%  load_hosts(search: input('hosts')).each_record do |host| -%>
```

Comment:
if you could s/<% /<%-/ here and on the line 16, 22 and 23, that would be great

Reply:
@ares this is an interesting point, when adding like below we can see new lines
```
<% ... %>
```
Then moving forward and keeping only like this, working as expected
```
<% ... -%>
```
However, when doing as below, looks like indifferent
```
<%- ... -%>
```

Insights about it?

Thank you

Reply:
@ares code updated according to your request.

Thank you!

Reply:
Just for the record <%- trims ghe whitespace left of the opening tag. Usually that is not needed, the opening tag starts on the beginning of the line. However it has the same length as <%=, so the inner code is nicely indented, while it does no harm.

Reply:
@ares perfect. Thanks for the explanation.